lib/stack_master/sparkle_formation/template_file.rb

```diff
@@ -71,7 +71,7 @@ def format
             newlines = lines.split("\n").map do |line|
               "#{line}#{newlines.pop}"
             end
-            if lines.starts_with?("\n")
+            if lines.start_with?("\n")
```

Comment:
Active support aliases Ruby's `start_with?` to `starts_with?`.

Let's just use the version that comes with Ruby.